src/main/java/org/jenkinsci/main/modules/sshd/SSHD.java

```diff
@@ -44,6 +48,12 @@
     private static final List<NamedFactory<Cipher>> ENABLED_CIPHERS = Arrays.<NamedFactory<Cipher>>asList(
         BuiltinCiphers.aes128ctr, BuiltinCiphers.aes192ctr, BuiltinCiphers.aes256ctr
     );
+
+    /**
+     * Comma-separated string of key exchange names to disable. Defaults to a list of DH SHA1 key exchanges, gets its value from {@link SystemProperties}.
+     */
+    private static final String EXCLUDED_KEY_EXCHANGES = SystemProperties.getString(SSHD.class.getName() + ".excludedKeyExchanges",
```

Comment:
Time to add README.md and document this option there?

Reply:
Is that conditional approval? I'm not writing docs if the entire thing gets shot down 😉

Reply:
FWIW I thought to document it on https://www.jenkins.io/doc/book/managing/system-properties/ because, for users, this is part of the core deliverable.